src/java.base/share/classes/sun/security/provider/certpath/RevocationChecker.java

```diff
@@ -488,10 +488,10 @@ static boolean isCausedByNetworkIssue(String type, CertStoreException cse) {
                 }
                 break;
             case "SSLServer":
-                result = (t != null && t instanceof IOException);
+                result = (t instanceof IOException);
```

Comment:
```suggestion
                result = t instanceof IOException;
```

Reply:
I like current approach with parentheses when `instanceof` is involved. Especially with assignment. In this case you can clearly see start and end of expression which would be assigned to variable.
@seanjmullan what do you think?